float64 timestamp

PerceptionObstacle perception_obstacle

TrajectoryPoint[] trajectory

bool is_static
